src/commands/build.ts

```diff
@@ -19,6 +19,13 @@ $ brew install ghostscript
 
 Ubuntu:
 $ apt-get install ghostscript
+
+Windows:
+    Download and install the Ghostscript for Windows at
+    https://www.ghostscript.com/download/gsdnld.html
+    Then, add the installed Ghostscript's bin directory
+    (e.g., "C:\\Program Files\\gs\\gs9.52\\bin")
+    to the PATH environment variable.
```

Comment:
Make the indent of **Ubuntu** and **macOS** section to be aligned as well as the Windows section.